strings/traumas.json

```diff
@@ -89,14 +89,25 @@
 		"I AM NORMAL. I CAN BE TRUSTED WITH INDUSTRIAL CHEMICALS",
 		"Welcome to the bathroom",
 		"THE AMOUNGS BROKE ALL MY BONES",
-		"HOLY SHIT IT'S @pick(john) @pick(factions)",
+		"HOLY SHIT IT'S @pick(john) @pick(factions)!!",
 		"BLrobo BLEEBUS.....",
 		"drugs are funny because they add new rp elements",
 		"ANOTHER SHIP... STERALIZED...",
 		"HOpeLes WAsN'T ALwAYS NaMeD HoPelESs BeFORE THe incIDENST...",
 		"@pick(semicolon)HopeLSS WAS ACTUAL. Ly a LIVIng WaePON....",
 		"@pick(semicolon)WHEN SHIPQUEST????? PANEL NEW??????",
-		"WHER.E SHIPQUEST???,???"
+		"WHER.E SHIPQUEST???,???",
+		"ei,,ither wAy... It Is WHt ti is",
+		"butT ShIPPtSt nEVER was a WrAZnOe liKE thaEt!!",
+		"A deEoP-setED dISEuire... fuR pERfECTiOsM...",
+		"HUMORER IS A DECLIAT THINGE!!",
+		"who's the asshole flying the pill class",
+		"IVOR@pick(y_replacements) WHAT THE FOUCKE ARE U DUING!!!??",
+		"RESIEST BIG MOETH",
+		"WE MUSTE RIASE UOP AGANST BEEG MOTNH",
```

Comment:
this reminds me from when i was a shitter in the shiptest discord

the glory days